docs/en/metadata_aa.rst

```diff
@@ -0,0 +1,101 @@
+.. include:: ./common_definitions.rst
+
+
+Attribute Authority Metadata
+++++++++++++++++++++++++++++
+
+An AA MUST publish in its EC a Metadata with *federation_entity*, a Metadata with *oauth_authorization_server* 
```

Comment:
rephrase and also align the it version please

An AA MUST publish in its EC a *federation_entity* Metadata and an *oauth_resource* Metadata, if the resources are protected it MUST also publish an *oauth_authorization_server* Metadata.